CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### PaymentSheet
 * [CHANGED][7144](https://github.com/stripe/stripe-android/pull/7144) PaymentSheet now features rounded corners with the radius provided in `PaymentSheet.Shapes.cornerRadiusDp`.
 * [FIXED][7190](https://github.com/stripe/stripe-android/pull/7190) Fixed an issue where amounts in Laotian Kip were displayed incorrectly.
+* [ADDED] Support for AmazonPay (private beta) with PaymentIntents.
```

Comment:
Nit: To align with the addition of GrabPay in the same release, we should word them the same. What do you think of this?

```suggestion
* [ADDED][7199](https://github.com/stripe/stripe-android/pull/7199) PaymentSheet now supports AmazonPay for PaymentIntents in private beta.
```

We should also move it to the top, so that we have `ADDED` > `CHANGED` > `FIXED`.